File: src/Exception/InvalidArithmeticOperatorException.java
package Exception;

/**
 * Created by vladc on 28.10.2016.
 */
public class InvalidArithmeticOperatorException extends ExpressionEvaluationException {
    public InvalidArithmeticOperatorException() {
        super();
    }
    public InvalidArithmeticOperatorException(String message) {
        super(message);
    }
}

File: Toy-Language-Interpreter_C#/Toy-Language-Interpreter/MyException/InvalidArithmeticOperatorException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Toy_Language_Interpreter.MyException
{
    class InvalidArithmeticOperatorException : ExpressionEvaluationException
    {
        public InvalidArithmeticOperatorException() { }
        public InvalidArithmeticOperatorException(string msg) : base(msg) { }
        public InvalidArithmeticOperatorException(string msg, Exception e) : base(msg, e) { }
    }
}
